src/adapter/src/coord/appends.rs

```diff
@@ -363,21 +289,24 @@ impl<S: Append + 'static> Coordinator<S> {
 
     /// Enqueue requests to advance all local inputs (tables) to the current wall
     /// clock or at least a time greater than any previous table read (if wall
-    /// clock has gone backward). These are not processed in a single append call
-    /// because they currently processed serially by persist. In order to allow
-    /// other coordinator messages to be processed (like user queries), split up the
-    /// processing of this work.
+    /// clock has gone backward).
     pub(crate) async fn queue_local_input_advances(&mut self, advance_to: mz_repr::Timestamp) {
-        self.advance_tables.insert(
-            advance_to,
-            self.catalog.entries().filter_map(|e| {
-                if e.is_table() || e.is_storage_collection() {
-                    Some(e.id())
-                } else {
-                    None
-                }
-            }),
-        );
+        self.internal_cmd_tx
```

Comment:
We might want to just send the advancement directly here instead of sending this message and doing it later. Either way works though.

Reply:
I chose to do it this way because this function is called after the message processing, and it seems that we should maintain the property that all actual work is done via messages.